core/tst/CleanUpS3.kt

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
accident

Reply:
Grrr... I need to move that to an ignore file...